cdk/domino_cdk/eks_stack.py

```diff
@@ -298,11 +298,13 @@ def provision_eks_cluster(self):
         max_nodegroup_azs = self.config["eks"]["max_nodegroup_azs"]
 
         for name, cfg in self.config["eks"]["managed_nodegroups"].items():
-            cfg["labels"] = {**cfg["labels"], **self.config["eks"]["global_node_labels"]}
```

Comment:
fix of pre-existing bug. You cannot have labels with custom ami_id but you also cannot remove labels because of this line.